src/modules/vtol_att_control/standard.cpp

```diff
@@ -277,39 +233,38 @@ void Standard::update_vtol_state()
 void Standard::update_transition_state()
 {
 	float mc_weight = 1.0f;
+	float time_now = (float)hrt_absolute_time();
```

Comment:
Why are you keeping this as a float?

Reply:
@dagar Are you only asking about time_now or also about time_since_trans_start below?
You are right, I don't need to have this as a float, I can cast it to a float in the computation of time_since_trans_start.
Not sure if you were debating about storing values as float vs integers.

Reply:
I mean keep it as a uint64 if working in microseconds, or convert to seconds and keep it as a float. For everything in this VTOL controller seconds is good enough, and easier to work with (fits with parameters).